### PR TITLE
Remove ab3

### PR DESCRIPTION
**Pull request by Harry-Gao-H**

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/MrTechRecruiterParserTest.java [hunk 1]
```diff
@@ -1,38 +1,12 @@
...
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
-import seedu.address.model.person.Person;
-import seedu.address.testutil.EditPersonDescriptorBuilder;
-import seedu.address.testutil.PersonBuilder;
-import seedu.address.testutil.PersonUtil;
-
 public class MrTechRecruiterParserTest {
```
Comment (LMAOboxhack) [inline src/test/java/seedu/address/logic/parser/MrTechRecruiterParserTest.java:3]: Added #174.

File: src/test/java/seedu/address/logic/parser/ParserUtilTest.java [hunk 2]
```diff
@@ -1,24 +1,14 @@
...
 
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Phone;
-import seedu.address.model.tag.Tag;
+
 
 public class ParserUtilTest {
```
Comment (LMAOboxhack) [inline src/test/java/seedu/address/logic/parser/ParserUtilTest.java:13]: Added #175.

File: src/test/java/seedu/address/storage/JsonUserPrefsStorageTest.java [hunk 3]
```diff
@@ -49,19 +49,23 @@ private Path addToTestDataPathIfNotNull(String userPrefsFileInTestDataFolder) {
                 : null;
```
Comment (LMAOboxhack) [inline src/test/java/seedu/address/storage/JsonUserPrefsStorageTest.java:49]: Added #176.

File: src/test/java/seedu/address/storage/StorageManagerTest.java [hunk 4]
```diff
@@ -1,8 +1,6 @@
 package seedu.address.storage;
```
Comment (LMAOboxhack) [inline src/test/java/seedu/address/storage/StorageManagerTest.java:1]: Added #177.

File: src/test/java/seedu/address/testutil/TestUtil.java [hunk 5]
```diff
@@ -33,23 +32,18 @@ public static Path getFilePathInSandboxFolder(String fileName) {
...
+        return Index.fromOneBased(model.getFilteredPositionList().size());
     }
 
-    /**
-     * Returns the person in the {@code model}'s person list at {@code index}.
-     */
-    public static Person getPerson(Model model, Index index) {
-        return model.getFilteredPersonList().get(index.getZeroBased());
-    }
+
```
Comment (LMAOboxhack) [inline src/test/java/seedu/address/testutil/TestUtil.java:48]: Perhaps we should split these methods into two? e.g. `getPositionMidIndex()` and `getApplicantMidIndex()`

File: src/test/java/seedu/address/testutil/TestUtil.java [hunk 6]
```diff
@@ -33,23 +32,18 @@ public static Path getFilePathInSandboxFolder(String fileName) {
     }
 
     /**
-     * Returns the middle index of the person in the {@code model}'s person list.
+     * Returns the middle index of the person in the {@code model}'s position list.
```
Comment (LMAOboxhack) [inline src/test/java/seedu/address/testutil/TestUtil.java:35]:
```suggestion
     * Returns the middle index of the position in the {@code model}'s position list.
```

File: src/test/java/seedu/address/testutil/TestUtil.java [hunk 7]
```diff
@@ -33,23 +32,18 @@ public static Path getFilePathInSandboxFolder(String fileName) {
...
      */
     public static Index getMidIndex(Model model) {
-        return Index.fromOneBased(model.getFilteredPersonList().size() / 2);
+        return Index.fromOneBased(model.getFilteredPositionList().size() / 2);
     }
 
     /**
-     * Returns the last index of the person in the {@code model}'s person list.
+     * Returns the last index of the person in the {@code model}'s position list.
```
Comment (LMAOboxhack) [inline src/test/java/seedu/address/testutil/TestUtil.java:42]:
```suggestion
     * Returns the last index of the position in the {@code model}'s position list.
```

**Review by LMAOboxhack — COMMENTED** [review]
Minor nits, otherwise LGTM!

**Review by BenjaminTan99 — APPROVED** [review]
LGTM. If resolving bloated code can be done in another PR.

File: src/test/java/seedu/address/testutil/TestUtil.java (hunk 5 above)
Comment (Harry-Gao-H) [reply src/test/java/seedu/address/testutil/TestUtil.java]: Yes i agree. This can be changed later when test cases are added.

**Pull request merged**